PhysicsTools/NanoAOD/python/jetsAK4_Puppi_cff.py

```diff
@@ -113,7 +114,10 @@
 
 run2_nanoAOD_ANY.toModify(
     jetPuppiTable.variables,
-    btagCSVV2 = Var("bDiscriminator('pfCombinedInclusiveSecondaryVertexV2BJetTags')",float,doc=" pfCombinedInclusiveSecondaryVertexV2 b-tag discriminator (aka CSVV2)",precision=10)
+    btagCSVV2 = Var("bDiscriminator('pfCombinedInclusiveSecondaryVertexV2BJetTags')",float,doc=" pfCombinedInclusiveSecondaryVertexV2 b-tag discriminator (aka CSVV2)",precision=10),
```

Comment:
Is keeping btagCSVV2 intended? I understood these were going to be dropped.

Reply:
Shall we drop them also from the run2 modifiers? For CSVv2 this particular line actually does not change the behavior, it's only listed by git in the diff because a "comma" is added at the end of line, as more than just CSVv2 is dropped for Run3.

Reply:
Sorry Annika, I again got confused. It makes sense to leave it as it is, although we can consider dropping cvs althogether for Run2 as well later on.

Reply:
Ok, that's fine with me as well. But I guess dropping it entirely at one place in the code will force us to also introduce several more changes in other places, which we can safely postpone to focus on getting the changes for Run3 implemented. Just trying to avoid we run into problems with IBs if it's currently not strictly necessary.